Automattic/jetpack
# Try updating to React 15 and newer dops-components

As part of a separate project, I wanted to upgrade dops-components and its dependencies and build system, including an update to React 15 (to match Calypso)

This patch tries to do the smallest amount of the work so that Jetpack's UI still builds and works.

To test:

- checkout dops-components master (this [PR](https://github.com/Automattic/dops-components/pull/109) was just merged)
- cd dops-components and run `npm install` and `npm link`
- cd jetpack and run `npm link @automattic/dops-components`
- `gulp react:build`

Click around the UI. This is a cross-cutting change that could affect every React component. Hopefully everything works just fine.

This PR also fixes a couple of warnings I was getting in the browser. Not sure if they're new or not. e.g. enable-background prop in React SVG elements should be enableBackground.
